packages/suite/src/components/settings/SettingsLayout/index.tsx

```diff
@@ -7,6 +7,7 @@ const Wrapper = styled.div`
     display: flex;
     flex: 1;
     flex-direction: column;
+    max-width: 768px;
```

Comment:
let's take this from global variables

```
import { variables } from '@trezor/components';
```

then use `variables.SCREEN_SIZE.MD`

Reply:
done, rebased